monotime/example_test.go

```diff
@@ -0,0 +1,14 @@
+package monotime_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aristanetworks/goarista/monotime"
+)
+
+func Example() {
```

Comment:
I added a separate test - you can see how this function looks on GDDO here: https://godoc.org/github.com/kevinburke/goarista/monotime#example-package

If I switched this to only be a test we'd lose the example